### DISCUSSION TITLE
set urlLoad in storage manager

### DISCUSSION THREAD

**Discussion by iabhiyaan** [GrapesJS/grapesjs#3848] [Q&A]
Hello @artf, Is there any way to reload different template and replace previously rendered template based on click event ?  

Scenario:

At initial render,
in `mounted()` we call 
```js
grapesjs.init({
 storageManager: {
   urlLoad: loadtemplate-endpoint/1
 }
}) 

```

How to load template again but for some other events like clicking button
urlLoad: loadtemplate-endpoint/2

**Comment by artf** [comment]
Check the API reference https://grapesjs.com/docs/api/editor.html#load

**Reply by iabhiyaan** [reply]
Thankyou for your response
